packages/terra-paginator/src/ControlledPaginator.jsx

```diff
@@ -262,7 +273,12 @@ class Paginator extends React.Component {
   }
 
   render() {
-    return <ResponsiveElement tiny={this.reducedPaginator()} small={this.defaultPaginator()} />;
+    return (
+      <ResponsiveElement onChange={value => this.setState({ breakpoint: value })}>
```

Comment:
Should we do something like the following so we only update state when transitioning between tiny and non-tiny?
```js
<ResponsiveElement
    onChange={value => {
        const showReducedPaginator = value === 'tiny';

        if (this.state.showReducedPaginator !== showReducedPaginator) {
            this.setState({ showReducedPaginator })
        }
    }}
>
    { this.state.showReducedPaginator ? this.reducedPaginator() : this.defaultPaginator()}
</ResponsiveElement>
```

Reply:
Fixed here 93f9aa9.